test/Microsoft.IdentityModel.TestUtils/TestSets.cs

```diff
@@ -883,7 +884,7 @@ public static SignedInfoTestSet ReferenceDigestValueNotBase64
                                 SecurityAlgorithms.EnvelopedSignature,
                                 SecurityAlgorithms.ExclusiveC14n,
                                 Default.ReferenceDigestMethod,
-                                digestValue))
+                                digestValue, typeAttr))
```

Comment:
Super nit: following the style of the lines above
```suggestion
                                digestValue, 
                                typeAttr))
```

Reply:
Thank you so much. I will resolve it